tests/prog_translator_tests.cc

```diff
@@ -243,7 +243,7 @@ TEST(ProgTranslatorTests, translation) {
   EXPECT_NO_THROW(SolveProg(iface));
   const double* soln = iface->getColSolution();
   const double* check = checkface.getColSolution();
-  array_double_eq(soln, check, narcs + nfaux, "soln");
+  array_double_near(soln, check, narcs + nfaux, 0.00000000001, "soln");
```

Comment:
This margin was chosen arbitrarily

Reply:
I think there is a cyclus definition of small somewhere - maybe `eps`?  Not sure if/how it's available in testing

Reply:
`cy_eps`

Reply:
Had to include `cyc_limits.h`, now we pass in cy_eps as the margin